recipes/cloudpickle/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: cloudpickle
+  version: {{ version }}
+
+source:
+  fn: cloudpickle-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/c/cloudpickle/cloudpickle-{{ version }}.tar.gz
+  md5: d6c92acdfa253a50b2ef86a1b6443e39
+
+build:
+  number: 0
+
```

Comment:
cloudpickle can use setuptools, so you can add

`script: python setup.py install --single-version-externally-managed --record record.txt`